packages/gamut/src/Form/__tests__/Checkbox-test.tsx

```diff
@@ -10,10 +10,14 @@ describe('<Checkbox>', () => {
   };
 
   it('sets the input checked state when the prop is passed', () => {
-    const wrapper = shallow(<Checkbox {...defaultProps} checked />);
-    expect(wrapper.find('input[type="checkbox"]').prop('checked')).toEqual(
-      true
+    const onChangeCallback = jest.fn();
```

Comment:
Teeny tiny non-blocking nit: "callback" is implied by the "on change" name, type being a function, and its usage as a function prop.

```suggestion
    const onChange = jest.fn();
```